.github/actions/free-up-disk-space/action.yml

```diff
@@ -23,12 +23,30 @@ runs:
         sudo rm -rf /usr/share/swift
 
         echo "Disk usage after cleanup:"
-        df -h
+        df -hT
 
     - name: Prune docker images
       shell: bash
       run: |
         echo "Pruning docker images"
         docker image prune -a -f
         docker system df
-        df -h
+        df -hT
+
+    - name: Move docker data directory
+      shell: bash
+      run: |
+        echo "Stopping docker service ..."
+        sudo systemctl stop docker
+        DOCKER_DEFAULT_ROOT_DIR=/var/lib/docker
+        DOCKER_ROOT_DIR=/mnt/docker
```

Comment:
I wonder if there are better ways to configure beforehand without restarting docker

Reply:
Discussed with @sivanantha321, configuring daemon.json does not work with minikube. Also https://github.com/moby/moby/issues/3127 for reference